jwilson.js

```diff
@@ -0,0 +1 @@
+console.log('Something, it could be anything.');
```

Comment:
This is a comment. I am commenting on this line. Feel free to feel commented on.